www/__tests__/useImperialConfig.test.ts

```diff
@@ -53,3 +61,15 @@ describe('convertSpeed', () => {
     expect(convertSpeed(6.7056, true)).toBeCloseTo(15); // Approximately 15 mph
   });
 });
+
+describe('useImperialConfig', () => {
+  it('returns ImperialConfig with imperial units', () => {
```

Comment:
Imperial units are the old ones (popularized by the British Empire), so `miles`, `mph` etc. These are non-imperial or SI units. Shouldn't we also add a similar check where we expect to see `miles`, `mph` etc?